opencti-platform/opencti-graphql/src/domain/user.js

```diff
@@ -542,6 +548,86 @@ export const checkPasswordFromPolicy = async (context, password) => {
   }
 };
 
+export const sendEmailToUser = async (context, user, input) => {
```

Comment:
I wonder about moving that to emailTemplate domain.... it's not 100% accurate but I'm so sad when we continue to make this file bigger and bigger.